server/services/licenceService.ts

```diff
@@ -149,6 +149,14 @@ export default class LicenceService {
     return this.licenceApiClient.createLicence(licence, user)
   }
 
+  private async getCRONumber(prisonerNumber: string, user: User): Promise<string> {
+    const prisoners = await this.prisonerService.searchPrisonersByNomisIds([prisonerNumber], user)
+    if (prisoners != null) {
+      return prisoners[0].croNumber
+    }
+    return ''
```

Comment:
We've already talked about it off GitHub, but this should just be `return null`. We don't want to be setting it to an empty string if it's not there, or it will look odd in the service and on the printed PDFs